Background mobile

tanx opened the pull request:
Closes #520, closes #485

valentinewallace commented on src/asset/img/textured-bg-mobile.svg:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
I don't quite understand why there's a `.js` and ` .svg` file for every background..

tanx replied:
Because we might need the source .svg in the future and someone might not have access to sketch e.g. not on macOS.

tanx commented on package.json:
```diff
@@ -24,7 +24,8 @@
     "storybook": "start-storybook -p 9009 -s public",
     "build-storybook": "build-storybook -s public",
-    "build-icon": "svgr --native --icon --filename-case kebab -d src/asset/icon src/asset/icon"
+    "build-icon": "svgr --native --icon --filename-case kebab -d src/asset/icon src/asset/icon",
```
lol yeah, programming isn't very vegan friendly 🥙

tanx commented:
> Had a few questions and one suggestion but I approve this PR to merge :)

@valentinewallace I rebased on master and refactored the `BackgroundImage` like you suggested in https://github.com/lightninglabs/lightning-app/pull/534/commits/c61aae61d1050bece833334154359cbe06b40492

Going ahead with merging this for now.
